addons/website_event_meet/tests/test_website_event_meet.py

```diff
@@ -54,6 +56,8 @@ def test_meeting_room_copy(self):
 
         meeting_room_3 = meeting_room_1.copy()
         self.assertEqual(meeting_room_3.room_name, 'odoo-room-test-meeting-room-2')
+        # Ensure room_max_capacity is copied to new meeting room
+        self.assertEqual(meeting_room_3.room_max_capacity, "20")
```

Comment:
Could be great to check that ``max_capacity`` on linked ``chat.room`` is also 20

Reply:
I have updated this one